styles/nixos-manual/catppuccin.user.css

```diff
@@ -0,0 +1,193 @@
+  #catppuccin(@lookup, @accent) {
+    @rosewater: @catppuccin[@@lookup][@rosewater];
+    @flamingo: @catppuccin[@@lookup][@flamingo];
+    @pink: @catppuccin[@@lookup][@pink];
+    @mauve: @catppuccin[@@lookup][@mauve];
+    @red: @catppuccin[@@lookup][@red];
+    @maroon: @catppuccin[@@lookup][@maroon];
+    @peach: @catppuccin[@@lookup][@peach];
+    @yellow: @catppuccin[@@lookup][@yellow];
+    @green: @catppuccin[@@lookup][@green];
+    @teal: @catppuccin[@@lookup][@teal];
+    @sky: @catppuccin[@@lookup][@sky];
+    @sapphire: @catppuccin[@@lookup][@sapphire];
+    @blue: @catppuccin[@@lookup][@blue];
+    @lavender: @catppuccin[@@lookup][@lavender];
+    @text: @catppuccin[@@lookup][@text];
+    @subtext1: @catppuccin[@@lookup][@subtext1];
+    @subtext0: @catppuccin[@@lookup][@subtext0];
+    @overlay2: @catppuccin[@@lookup][@overlay2];
+    @overlay1: @catppuccin[@@lookup][@overlay1];
+    @overlay0: @catppuccin[@@lookup][@overlay0];
+    @surface2: @catppuccin[@@lookup][@surface2];
+    @surface1: @catppuccin[@@lookup][@surface1];
+    @surface0: @catppuccin[@@lookup][@surface0];
+    @base: @catppuccin[@@lookup][@base];
+    @mantle: @catppuccin[@@lookup][@mantle];
+    @crust: @catppuccin[@@lookup][@crust];
+    @accent-color: @catppuccin[@@lookup][@@accent];
+
+    --ctp-rosewater: @rosewater;
+    --ctp-flamingo: @flamingo;
+    --ctp-pink: @pink;
+    --ctp-mauve: @mauve;
+    --ctp-red: @red;
+    --ctp-maroon: @maroon;
+    --ctp-peach: @peach;
+    --ctp-yellow: @yellow;
+    --ctp-green: @green;
+    --ctp-teal: @teal;
+    --ctp-sky: @sky;
+    --ctp-sapphire: @sapphire;
+    --ctp-blue: @blue;
+    --ctp-lavender: @lavender;
+    --ctp-text: @text;
+    --ctp-subtext1: @subtext1;
+    --ctp-subtext0: @subtext0;
+    --ctp-overlay2: @overlay2;
+    --ctp-overlay1: @overlay1;
+    --ctp-overlay0: @overlay0;
+    --ctp-surface2: @surface2;
+    --ctp-surface1: @surface1;
+    --ctp-surface0: @surface0;
+    --ctp-base: @base;
+    --ctp-mantle: @mantle;
+    --ctp-crust: @crust;
+
+    color-scheme: if(@lookup = latte, light, dark);
+
+    ::selection {
+      background-color: fade(@accent-color, 30%);
+    }
+
+    body {
+      background-color: @base !important;
+      color: @text;
+    }
+
+    p,
+    ol,
+    ul {
+      color: @text !important;
+    }
+
+    a {
+      border-bottom-color: @accent-color !important;
+      color: @accent-color;
+    }
+
+    h1,
+    h2,
+    h3,
+    h4 {
+      color: @accent-color;
+    }
+    h5 {
+      color: fade(@accent-color, 85);
+    }
+
+    hr,
+    div.toc {
+      border-top-color: transparent;
+      border-bottom-color: @accent-color !important;
+    }
+
+    div.variablelist {
+      color: @text;
+    }
+
+    code {
+      &.programlisting.hljs {
+        background-color: fade(@blue, 5) !important;
+      }
+    }
+
+    table.informaltable {
+      border-color: @surface1;
+      thead {
+        background-color: @mantle;
+      }
+    }
+
+    .note,
+    .tip {
```

Comment:
Despite the original styling using the same colors for notes and tips, I believe we should differentiate between the two by using green for tips (keep using blue for notes separately).

Reply:
I agree, I was original thinking doing this, but decided to just keep it as-is in the first rendition.

Reply:
With regards this separation, do you think changing caution vs warning should be done too?
They are currently both `@peach`.

Example using `@yellow` for Caution while keeping warning as `@peach`
![image](https://github.com/user-attachments/assets/9edfd4c6-3211-4287-a0f6-f2badd76b55a)
![image](https://github.com/user-attachments/assets/80abee6b-ae21-4b74-8cba-6a0e6592e872)


Here is also an example for Tip vs Note
![image](https://github.com/user-attachments/assets/ce9f07d4-b9d6-4af7-86ae-d7a9ed6d96d5)
![image](https://github.com/user-attachments/assets/7e58ed38-87a6-42f1-9c77-840feae273c3)

Reply:
Yep that looks great! Let's do different colors for all 4 like you showed.